### PR TITLE
Enhancements to user guide analyzing content

### PR DESCRIPTION
**Pull request by michelle-purcell**
**Description:** As a Red Hat writer, I need information on the different ways you can use Vale

As part of the vale-at-red-hat working user group initiative and creation of [Installing the Vale Client with the vale-at-red-hat rules](https://docs.google.com/document/d/1S9jZc2EZ8vUm33t0rrTEr6FZH0WnOXEs7ajcbQIaqv0/edit#), we have identified improvements and enhancements to the existing vale-at-red-hat [User Guide](https://redhat-documentation.github.io/vale-at-red-hat/docs/user-guide/introduction/).

**Highlights:**

* Updated nav to add a sub-section for grouping content that is about how to analyze your content with Vale.
* Migrated content from [vale-at-red-hat rules](https://docs.google.com/document/d/1S9jZc2EZ8vUm33t0rrTEr6FZH0WnOXEs7ajcbQIaqv0/edit#):
* Added a new topic to introduce the Red Hat Vale web app.

**Additional context**
As discussed in the [vale-at-red-hat slack channel ](https://app.slack.com/client/T027F3GAJ/C0218RXJK5E)and recent open office hours workgroup to provide a CCS vale-at-red-hat onboarding toolkit.

### CODE REVIEW

File: modules/user-guide/pages/analyze-vale-app.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+* To use the Vale starter app, enter https://vale.vercel.app/ in any browser, paste your AsciiDoc content, and click *Lint*.
+
+* By default, the Red Hat Vale web app displays warnings and errors only. You can also display suggestions by using options menu on the left. 
+
+////
+
+[IMPORTANT]
+====
+Add disclaimer about the support of the app here.
```
Comment (michelle-purcell) [inline modules/user-guide/pages/analyze-vale-app.adoc:26]: @rolfedh - Do you have any suggestions for providing guidance about the supportability of the app?

File: modules/user-guide/nav.adoc [hunk 2]
```diff
@@ -2,12 +2,14 @@
 
 * xref:introduction.adoc[]
 * xref:installing-vale-cli.adoc[]
-* xref:linting-vale-cli.adoc[]
-* xref:using-vale-in-the-ide.adoc[]
+* xref:analyze-with-vale.adoc[]
+** xref:analyze-vale-app.adoc[]
```
Comment (michelle-purcell) [inline modules/user-guide/nav.adoc:6]: @themr0c - 
Does the doc navigation support nested topics? 
If not I will go back to the drawing board on how to restructure the content.

File: modules/user-guide/pages/analyze-vale-app.adoc [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+With the web app, you can run Vale on one source file at a time. You can also analyze a section of a content file.
+
+* To use the Vale starter app, enter https://vale.vercel.app/ in any browser, paste your AsciiDoc content, and click *Lint*.
+
+* By default, the Red Hat Vale web app displays warnings and errors only. You can also display suggestions by using options menu on the left. 
+
+////
+[IMPORTANT]
+====
+Add disclaimer about the support of the app here.
```
Comment (rolfedh) [inline modules/user-guide/pages/analyze-vale-app.adoc:25]:
```suggestion
[NOTE]
====
The [vale.vercel.com](https://vale.vercel.app/) web app is graciously hosted and funded by an individual Red Hatter for low-volume use. For regular production use, please install and run Vale with the RedHat style on your local machine. 
```

**Comment by michelle-purcell** [comment]
@tmalove & @rolfedh 
This PR contains the next iteration of edits to the docs to migrate the google doc content over.
This is not urgent. I'm not sure if the structural changes to the nav will work.
Please review as and when. Thank you :-)

File: modules/user-guide/pages/analyze-vale-app.adoc [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+[id="proc_linting-vale-cli_{context}"]
+= Analyzing your content by using the Red Hat Vale web app
+
+If you are new to Vale and want to quickly get started without installing the client and prerequisites, use the web-based version of Vale. The Red Hat web app is preconfigured to analyze your content by using Red Hat style rules. The `vale-at-red-hat`` rules are built in and there is no need to install and maintain the Vale client and the `vale-at-red-hat`` style rules. 
+
+With the web app, you can run Vale on one source file at a time. You can also analyze a section of a content file.
+
+* To use the Vale starter app, enter https://vale.vercel.app/ in any browser, paste your AsciiDoc content, and click *Lint*.
+
+* By default, the Red Hat Vale web app displays warnings and errors only. You can also display suggestions by using options menu on the left.
```
Comment (rolfedh) [inline modules/user-guide/pages/analyze-vale-app.adoc:20]:
```suggestion
* By default, the Red Hat Vale web app displays warnings and errors only. You can also display suggestions by using the options menu on the left. 
```

File: modules/user-guide/pages/analyze-vale-app.adoc [hunk 5]
```diff
@@ -0,0 +1,27 @@
+// Metadata for Antora
+:navtitle: Analyzing with the Red Hat Vale app
+:keywords: vale, app
+:description: Describes how to quickly get started with Vale and analyze a batch of content by using the Vale app.
```
Comment (rolfedh) [inline modules/user-guide/pages/analyze-vale-app.adoc:4]:
```suggestion
:description: Quickly analyze content by using the Red Hat Vale app online. No installation required.
```

File: modules/user-guide/pages/analyze-vale-app.adoc [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
```
Comment (rolfedh) [inline modules/user-guide/pages/analyze-vale-app.adoc:4]:
```suggestion
:description: Quickly get started with Vale and analyze a batch of content by using the Vale app.
```

File: modules/user-guide/pages/analyze-vale-app.adoc [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+:page-aliases: end-user-guide:get-started-vale-app.adoc
+// End of metadata for Antora
+// Metadata for Modular Docs
+:context: assembly_getting-started-with-vale
+:_module-type: PROCEDURE
+// End of metadata for Modular Docs
+[id="proc_linting-vale-cli_{context}"]
+= Analyzing your content by using the Red Hat Vale web app
+
+If you are new to Vale and want to quickly get started without installing the client and prerequisites, use the web-based version of Vale. The Red Hat web app is preconfigured to analyze your content by using Red Hat style rules. The `vale-at-red-hat`` rules are built in and there is no need to install and maintain the Vale client and the `vale-at-red-hat`` style rules.
```
Comment (rolfedh) [inline modules/user-guide/pages/analyze-vale-app.adoc:14]:
```suggestion
To quickly get started, you can use a web-based version of Vale with the Red Hat style rules.
```

File: modules/user-guide/pages/analyze-vale-app.adoc [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+// Metadata for Modular Docs
+:context: assembly_getting-started-with-vale
+:_module-type: PROCEDURE
+// End of metadata for Modular Docs
+[id="proc_linting-vale-cli_{context}"]
+= Analyzing your content by using the Red Hat Vale web app
+
+If you are new to Vale and want to quickly get started without installing the client and prerequisites, use the web-based version of Vale. The Red Hat web app is preconfigured to analyze your content by using Red Hat style rules. The `vale-at-red-hat`` rules are built in and there is no need to install and maintain the Vale client and the `vale-at-red-hat`` style rules. 
+
+With the web app, you can run Vale on one source file at a time. You can also analyze a section of a content file.
```
Comment (rolfedh) [inline modules/user-guide/pages/analyze-vale-app.adoc:16]:
```suggestion
```
Maybe fold this into the following step.

File: modules/user-guide/pages/analyze-vale-app.adoc [hunk 9]
```diff
@@ -0,0 +1,25 @@
+// Metadata for Antora
+:navtitle: Analyzing with the Red Hat Vale app
```
Comment (rolfedh) [inline modules/user-guide/pages/analyze-vale-app.adoc:2]:
```suggestion
:navtitle: Using Vale-at-Red-Hat online
```

Comment (rolfedh) [reply modules/user-guide/pages/analyze-vale-app.adoc:2]: Can we standardize/simplify by using the title of the app, "Vale-at-Red-Hat online", throughout?

File: modules/user-guide/pages/analyze-vale-app.adoc (hunk 7 above)
Comment (rolfedh) [inline modules/user-guide/pages/analyze-vale-app.adoc:14]:
```suggestion
To get familiar with Vale without installing the Vale software on your local machine, you can use Vale-at-Red-Hat online. This web app is preconfigured to analyze your content by using Red Hat's style rules.
```

File: modules/user-guide/pages/analyze-vale-app.adoc [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+:_module-type: PROCEDURE
+// End of metadata for Modular Docs
+[id="proc_linting-vale-cli_{context}"]
+= Analyzing your content by using the Red Hat Vale web app
+
+If you are new to Vale and want to quickly get started without installing the client and prerequisites, use the web-based version of Vale. The Red Hat web app is preconfigured to analyze your content by using Red Hat style rules. The `vale-at-red-hat`` rules are built in and there is no need to install and maintain the Vale client and the `vale-at-red-hat`` style rules. 
+
+With the web app, you can run Vale on one source file at a time. You can also analyze a section of a content file.
+
+* To use the Vale starter app, enter https://vale.vercel.app/ in any browser, paste your AsciiDoc content, and click *Lint*.
```
Comment (rolfedh) [inline modules/user-guide/pages/analyze-vale-app.adoc:18]:
```suggestion
* To use the Vale starter app, visit https://vale.vercel.app/ , paste your content or drop a content file in the app window, and click *Lint*.
```

File: modules/user-guide/pages/analyze-vale-app.adoc [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+
+* By default, the Red Hat Vale web app displays warnings and errors only. You can also display suggestions by using options menu on the left. 
+
+[NOTE]
+====
+The [vale.vercel.com](https://vale.vercel.app/) web app is graciously hosted and funded by an individual Red Hatter for low-volume use. For regular production use, please install and run Vale with the RedHat style on your local machine.
```
Comment (rolfedh) [inline modules/user-guide/pages/analyze-vale-app.adoc:24]:
```suggestion
The [vale.vercel.com](https://vale.vercel.app/) web app is graciously hosted and funded by an individual Red Hatter for low-volume use. For regular production use, please install and run Vale with the `RedHat` style on your local machine. 
```

**Comment by michelle-purcell** [comment]
@rolfedh - Thanks very much for your review :100: All comments are now fixed. I agree with all of your suggestions.
Any idea why the Linting build job fails in this PR? There's only 1 flag and that's a suggestion.

**Comment by rolfedh** [comment]
> Any idea why the Linting build job fails in this PR? There's only 1 flag and that's a suggestion.

I'm not certain. I pinged Fabrice to see what we might do about that.

**Pull request merged**